pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Inclusive Language style guide
+description: This style guide outlines the guideline for using inclusive languages in documentation.
+---
+
+# Inclusive Language
+
+While writing documentation we are indirectly reaching out to thousands of people all around the globe. 
+Thus, we must ensure that our diverse audience can connect with our information. 
+
+### 1. Culturally Inclusive Language
+
+Some phrases and words are popularly used only in specific regions of the world, thus our audience from other parts 
+of the world might need help to relate to the same. Hence, we should try to avoid using such phrases in our documentation. 
+Some examples of Region Specific Language are -
+- “It’s not rocket science” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+- Phrases written in regional languages instead of English.
+- “It’s not a piece of cake” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+
+### 2. Gender Neutral Language - 
+
+Some phrases and words are targeting men or men/women specifically. This might result in other gender groups feeling 
+left out while reading our documentation. 
+Some examples of Gender Neutral Language are -
+- Using “they/them” instead of “his/him” or “she/her”.
+- Using “Hello everyone” instead of “Hello guys”.
+- Using “Chairperson” instead of “Chairman” or “Chairwoman”.
+
+### 3. People First Language - 
+
+The placement of the word “People” matters a lot when it comes to writing documentation. 
+Using the word “People” in the beginning makes it more pleasant as the problem is not displayed in the beginning.
+Some examples of Gender Neutral Language are -
+- Using “People with Mental Instability” instead of “Mentally Unstable People”.
```

Comment:
```suggestion
- Using “People with a mental health condition” instead of “Mentally Unstable People”.
```

Ah... ironically, "**mentally unstable** people" is not an appropriate or respectful term to use. It can be stigmatizing and reinforces negative stereotypes about mental health. 

Here are better examples with more appropriate language:
- Person with a mental health condition
- Person with a psychiatric disability
- Person with mental illness
- Person with a mental health challenge
- Person experiencing a mental health crisis